libs/scripts/scripts.js

```diff
@@ -22,11 +22,20 @@ import locales from '../utils/locales.js';
 const prodDomains = ['milo.adobe.com'];
 
 const stageDomainsMap = {
-  'www.adobe.com': 'www.stage.adobe.com',
-  'blog.adobe.com': 'blog.stage.adobe.com',
-  'business.adobe.com': 'business.stage.adobe.com',
-  'helpx.adobe.com': 'helpx.stage.adobe.com',
-  'news.adobe.com': 'news.stage.adobe.com',
+  'www.stage.adobe.com': { 'www.adobe.com': 'www.stage.adobe.com' },
+  'business.stage.adobe.com': {
```

Comment:
minor: can't we make just an array of prod domain to look for and then default replace function is .adobe.com -> .stage.adobe.com ? (and generate that mapping automatically once)

Reply:
I don't think this will work because we don't have an universal replacement for .adobe.com. It can be .stage.adobe.com or .hlx.page or .hlx.live.
Also Milo should be consumer agnostic so I would avoid having consumer specific rules, such as a replace function for .adobe.com -> .stage.adobe.com

Reply:
@npeltier - we thought about this in the previous iteration, but realised it's not something that would work.